SmartEVSE2.X/EVSE.c

```diff
@@ -130,14 +130,14 @@ unsigned int CalcCurrent();
 
 
 // Configuration settings
-#pragma	config FCMEN = OFF,	IESO = OFF, PRICLKEN = ON
+#pragma	config FCMEN = ON, IESO = ON, PRICLKEN = ON                             // Enable Internal/External Oscillator Switchover
 #pragma config PLLCFG = OFF, FOSC = HSMP                                        // High Speed Medium power (4-16Mhz), PLL Off
 #pragma	config BORV = 285, BOREN = ON, PWRTEN = ON
 #pragma	config WDTPS = 2048, WDTEN = OFF                                        // WDT timeout
 #pragma config CCP2MX = PORTB3, PBADEN = OFF, CCP3MX = PORTC6                   // PortB digital IO
 #pragma config HFOFST = OFF, T3CMX = PORTB5, P2BMX = PORTC0, MCLRE = INTMCLR
 
-#pragma config XINST = OFF, DEBUG = OFF, LVP = OFF, STVREN = ON
+#pragma config XINST = OFF, DEBUG = OFF, LVP = ON, STVREN = ON                  // Allow low voltage programming
```

Comment:
Enabling LVP will also enable the MCLR pin (which has no strong pull-up), so we have to leave this disabled.